python/ray/tune/examples/pbt_tune_cifar10_with_keras.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''Train keras CNN on the CIFAR10 small images dataset.
+
+The model comes from: https://zhuanlan.zhihu.com/p/29214791,
+and it gets to about 87% validation accuracy in 100 epochs.
+
+Note that the scipt cannot init cuda in parallel, hence it
```

Comment:
Can you tell me more about this?

Reply:
Yes. I write the script and run it on a local machine (with 2 GPU). And I find that all trials report "failed call to cuInit: CUDA_ERROR_NO_DEVICE". You can run the script and reproduce the problem.

Note that `tf.keras` shares a global graph and session, which might result in conflict when training in parallel in local machine.

Reply:
I think this would work if you also set `resources={"gpu": 1}` in the train_spec. By default, if GPU resources are requested then CUDA_VISIBLE_DEVICES will be set to the empty string which will disable GPU access.

Reply:
Good suggestion. I'd like to take a try later.

Reply:
Yeah, CUDA_VISIBLE_DEVICES will be assigned based on `ray.get_gpu_ids()`. So if you request 1 gpu it gets set to some gpu id, 2 gpus a list of two ids, 0 means the empty string.